src/sections/quiz/quiz-body.js

```diff
@@ -3,35 +3,119 @@ import {SvgIcon} from "@mui/material";
 import Container from "@mui/material/Container";
 import Box from "@mui/material/Box";
 import Logo from "../../components/svg-color/Logo.svg";
-import {Lora} from '@next/font/google';
-import Grid from '@mui/material/Grid';
+import {Lora} from "@next/font/google";
+import Grid from "@mui/material/Grid";
+import useSWR from "swr"
+import { useState } from "react";
+import { useSWRConfig } from "swr"
 
+const fetcher = (...args) => fetch(...args).then(res => res.json())
+const post_fetcher = (...args) => fetch(...args).then(res => res.json())
 const lora = Lora({preload: false});
 
-// function VariantButton(text) {
-function VariantButton() {
+
+function QuestionList() {
+  const { data, error, isLoading } = useSWR("http://localhost:8000/api/v1/quiz/questions/", fetcher);
+
+  if (error) return <div>Data loading error</div>
+  if (isLoading) return <div>Loading...</div>
+  return data;
+}
+
+function CreateQuiz(email) {
+  fetch(
+    "http://localhost:8000/api/v1/quiz/",
+    {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        email: email
+      })
+    }
+  )
+}
+
+function CreateAnswer(answer, question) {
+  if (question?.type == "radio") {
+    const response = fetch(
+      "http://localhost:8000/api/v1/quiz/answers/",
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          "Authorization": "BearerToken",
+        },
+        body: JSON.stringify({
+          question: question?.id,
+          choice: answer?.id,
+        })
+      }
+    ).then(response => {
+        if(response.status >= 400) {
+          throw new Error("Server responds with error! " + response.status);
+        } else if(response.status == 201) {
+          alert("Question answered successful");
```

Comment:
Same idea, show snackbar, informing, that question was successfully saved or do nothing to not distract user from a quiz.